distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Telemetry.enso

```diff
@@ -0,0 +1,15 @@
+private
+
+import Standard.Base.Nothing.Nothing
+import Standard.Base.Data.Text.Text
+import Standard.Base.Data.Vector.Vector
+
+polyglot java import org.slf4j.LoggerFactory
+
+
+log logger_name:Text msg:Text args:Vector -> Nothing =
```

Comment:
If we are going to have this method for every stdlib then I'm wondering if it really matters to include `logger_name` and not just hardcode it with libs' name.

Reply:
I believe it is still valuable to set a particular logger name inside a library.